### PR TITLE
Search loi by id 

### PR DESCRIPTION
**Pull request by t-my**

### CODE REVIEW

File: webapp/test/clj/lipas/backend/handler_test.clj [hunk 1]
```diff
@@ -199,7 +199,13 @@
...
     (is (= loi-category (:loi-category response-loi)))))
 
-
+(deftest get-loi-by-id
+  (let [{:keys [id] :as loi} (gen-loi!)
+        _    (core/index-loi! search loi :sync)
+        resp (app (-> (mock/request :get (str "/api/lois/" id))
+                      (mock/content-type "application/json")))
+        response-loi (<-json (:body resp))]
+    (is (= (dissoc loi :features) response-loi))))
```
Comment (vharmain) [inline webapp/test/clj/lipas/backend/handler_test.clj:208]: I think `:features` should match also

Comment (t-my) [reply webapp/test/clj/lipas/backend/handler_test.clj]: Removed.

File: webapp/src/clj/lipas/backend/handler.clj [hunk 2]
```diff
@@ -166,7 +166,18 @@
...
+              :body   (core/search-lois search query)}))}}
+
+      ["/:loi-id"
+      {:get
+       {:no-doc     false
+        :responses  {200 {:body :lipas.loi/document}}
+        :parameters {:path {:loi-id :lipas.loi/id}}
+        :handler
+        (fn [{:keys [parameters]}]
+          (println parameters)
```
Comment (vharmain) [inline webapp/src/clj/lipas/backend/handler.clj:178]: Please remove debug printing

Comment (t-my) [reply webapp/src/clj/lipas/backend/handler.clj]: removed

**Pull request merged**